internal/junitxml/report.go

```diff
@@ -229,3 +236,12 @@ func write(out io.Writer, suites JUnitTestSuites) error {
 	_, err = out.Write(doc)
 	return err
 }
+
+func appendFailFileLine(jtc *JUnitTestCase, failureOutput string) {
+	file, line, err := parser.ParseFailure(failureOutput)
+	if err != nil {
+		log.Warnf("Failed to parse failure output: %v", err)
```

Comment:
How about something like this?

```suggestion
		log.Warnf("Failed to parse file:line from test output: %v", err)
```

Reply:
addressed in https://github.com/asfaltboy/gotestsum/commit/937e59c6c6e49792492ef7a78aa84188b95e5fa6